test/test_autograd.py

```diff
@@ -2638,16 +2638,6 @@ def test_profiler(self):
             self.assertEqual(info.name, expected_name)
             last_end = info.cpu_interval.end
 
-    def test_record_function_callbacks(self):
-        x = torch.randn(10, 10)
-        with profile() as p:
-            with record_function("foo"):
-                y = x * 2 + 4
-
-        function_events = p.function_events
-        foo_event = [event for event in function_events if "foo" in event.name][0]
-        self.assertEqual(foo_event.count, 1)
-
```

Comment:
Why did we remove this test?

Reply:
Not sure, I had a merge conflict here so I may have resolved it incorrectly.